drivers/bridge/bridge.go

```diff
@@ -46,6 +42,8 @@ type Configuration struct {
 	EnableICC             bool
 	EnableIPForwarding    bool
 	AllowNonDefaultBridge bool
+	// Driver components
+	PortAllocator *portallocator.PortAllocator
```

Comment:
Since you are fixing the existing issues in portAllocator and portMapping, I would like to explore an option to make 
portAllocator and mapper as singleton.
Port Allocator and Port Mapper packages are designed to be managing and mapping the Host Port space and hence giving the ability to create multiple instances doesn't fit well.